inst/include/conditionalSampler.h

```diff
@@ -0,0 +1,673 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
+//
+// sampler.h: Rcpp integration of SMC library -- sampler object
```

Comment:
A pedantic point, but shouldn't this be conditionalSampler.h with a slughtly more descriptive name?

Reply:
Done! I think I understood the point about the first line so I won't remove it (for consistency with the other files in the package).